docs/source/reward_trainer.mdx

```diff
@@ -14,16 +14,22 @@ The [`RewardTrainer`] expects a very specific format for the dataset since the m
 
 Therefore the final dataset object should contain two 4 entries at least if you use the default [`RewardDataCollatorWithPadding`] data collator. The entries should be named:
 
-- `input_ids_chosen`
-- `attention_mask_chosen`
-- `input_ids_rejected`
-- `attention_mask_rejected`
+-   `input_ids_chosen`
+-   `attention_mask_chosen`
+-   `input_ids_rejected`
+-   `attention_mask_rejected`
 
 ## Using the `RewardTrainer`
 
-After preparing your dataset, you can use the [`RewardTrainer`] in the same way as the `Trainer` class from 🤗 Transformers. 
+<<<<<<< HEAD
+After standardizing your dataset, you can use the `RewardTrainer` as a classic Hugging Face Trainer.
+You should pass an `AutoModelForSequenceClassification` model to the `RewardTrainer`.
+=======
+After preparing your dataset, you can use the [`RewardTrainer`] in the same way as the `Trainer` class from 🤗 Transformers.
 You should pass an `AutoModelForSequenceClassification` model to the [`RewardTrainer`], along with a [`RewardConfig`] which configures the hyperparameters of the training.
 
+> > > > > > > origin/main
```

Comment:
Hmm there is a merge conflict here that has not been properly dealt with, can you please have a look 🙏

Reply:
Should be ready